docs/system-admin-guide/authentication/authentication-settings/README.md

```diff
@@ -66,6 +66,8 @@ There can be defined a number of other authentication settings.
 3. Enable or disable the **autologin option**. This allows a user to remain logged in, even if he/she leaves the site. If this option is activated, the “Stay signed in” option will appear on the login screen to be selected.
 4. Activate the **session expiration option**. If you select this option, an additional field will open, where you will be able to define the **inactivity time duration before the session expiry**.
 5. Define to **log user login, name, and mail address for all requests**.
-7. Do not forget to **save** your changes.
+6. Define a path to redirect users to after their first login. If left empty, users are redirected to the homepage.
```

Comment:
Maybe it makes sense to mention here that they get redirected to the homepage to see the onboarding tour?